main.js

```diff
@@ -2,9 +2,12 @@ class Color {
   constructor() {
     this.hexcode = this.#randomHexCode();
     this.locked = false;
-    this.id = Date.now().toString(); //id no work, fix me
+    this.id = Color.colorsMadeSoFar + 1;
```

Comment:
Do we want to use a different naming convention here? colorsMadeSoFar is a bit.......on the nose. What do you think? @aspitz1 @tfettig22

Reply:
I don't mind it being on the nose. It's easy to understand what it does.